parlai/mturk/core/mturk_data_handler.py

```diff
@@ -405,15 +426,36 @@ def log_start_task(self, worker_id, assignment_id, conversation_id):
             conn.commit()
 
     def log_award_amount(self, worker_id, assignment_id, amount, reason):
-        """Update a pairing state to add a bonus to be paid, appends reason"""
+        """Update a pairing state to add a task bonus to be paid,
+        appends reason. To be used for automatic evaluation bonuses
+        """
         with self.table_access_condition:
             conn = self._get_connection()
             c = conn.cursor()
             reason = "${} for {}\n".format(amount, reason)
+            # Bonus amount is stored in a cents int in the SQL table
+            cent_amount = int(amount * 100)
```

Comment:
do we enforce somewhere else that bonuses are set in dollar amounts?

edit:  i notice that is the case for the webapp, curious if this is still enforced elsewhere (or if that even matters for this particular function)

Reply:
We don't necessarily enforce it, though this call is only made from MTurkManager, and that API suggests the bonus amount is in dollars. The cent representation is only ever seen on the backend and it only exists for SQL storage reasons.